src/Front/Pages/SignupPage.razor.css

```
@@ -0,0 +1,207 @@
+
+/* span: mb-1 block text-sm text-slate-300 */
+.label-text {
+    display: block;
+    margin-bottom: 0.25rem;
+    font-size: 0.875rem;
+    line-height: 1.25rem;
+    color: #cbd5e1;
+}
+
+/* input: w-full rounded-md border border-slate-700 bg-slate-800 px-3 py-2 text-white outline-none focus:border-sky-500 */
+::deep .form-input {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: deep

(lint/correctness/noUnknownPseudoElement)